gms/application_state.cc

```diff
@@ -8,48 +8,57 @@
  */
```

Comment:
` code cleanup and improvements `

add `treewide: ` prefix to follow commit message conventions

Reply:
done